wolfcrypt/src/random.c

```diff
@@ -2627,8 +2627,12 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
 #elif defined(HAVE_RTP_SYS) || defined(EBSNET)
 
 #include "rtprand.h"   /* rtp_rand () */
+
+#if (defined(HAVE_RTP_SYS) || RTPLATFORM)
```

Comment:
We should not assume `RTPLATFORM` is always defined. Should this be `#if defined(HAVE_RTP_SYS) || defined(RTPLATFORM)`. Otherwise if `RTPLATFORM` is not defined it will be a compiler error. If `RTPLATFORM` is also an expression like `RTPLATFORM != 0` please add it explicit.